examples/motor-imagery/helpers/frequencybandselection_helpers.py

```diff
@@ -0,0 +1,226 @@
+"""
+=================================
+ Frequency Band Selection Helpers
+=================================
+
+This file contains helper functions for the frequency band selection example
+"""
+
+import numpy as np
+from mne import Epochs, events_from_annotations
+
+from pyriemann.estimation import Covariances
+from pyriemann.classification import class_distinctiveness
+
+
+def freq_selection_class_dis(raw, freq_band=[5., 35.], sub_band_width=4,
+                             sub_band_step=2, tmin=0.5, tmax=2.5,
+                             picks=None, event_id=None,
+                             cv=None, train_ind=None,
+                             method='cross_validation',
+                             return_class_dis=False):
+    r"""Select optimal frequency band based on class distinctiveness measure.
+
+        Optimal frequency band is selected by combining a filter bank with
+        a heuristic based on class distinctiveness on the manifold:
+        1. Filter training raw EEG data for each sub-band of a filter bank.
+        2. Estimate covariance matrices of filtered EEG for each sub-band.
+        3. Measure the class distinctiveness of each sub-band using the
+        classDis metric.
+        4. Find the optimal frequency band by starting from the sub-band
+        with the largest classDis and expanding the selected frequency band
+        as long as the classDis exceeds the threshold :math:`th`:
+
+        .. math::
+        th=(max(classDis)−min(classDis))×0.4
```

Comment:
This is actually because we assumed that selecting values falling into the top 40% of the classDis range is reasonable for selecting the frequency bands with relatively high classDis values for each subject.
Indeed, it is a good idea to generalize this parameter! I added the parameter named alpha.